### ISSUE TITLE
Controller action named Created does not generate.

### ISSUE THREAD

**Issue by MarkFl12** [T4MVC/R4MVC#152]
If I have an action on my controller called Created then I won't get anything generated for it by R4MVC.

**Comment by MarkFl12** [comment]
I suspect this is because there's a Created method on the Controller class for use in returning a Created status code result.

**Comment by btihanyi** [comment]
Yes this is because the same issue as [#128](https://github.com/T4MVC/R4MVC/issues/128#issuecomment-636319651).
